chatbot2.js

```diff
@@ -553,7 +553,7 @@ client.on("chat", (channel, userstate, message, self) => {
       userstate["mod"] ||
       userstate["username"] == config.credentials.channelName
     ) {
-      if (ttsLangs[args[0]] == undefined) {
+      if (ttsLangs[args[0]] == undefined && !!args[0] && !!args[1]) {
```

Comment:
double "!", what it does? PepoG

Reply:
inverting a bool 2 times should work with args[0] as well
sometimes you also do a double bool invert conversion if you wanna get a type
like string converted to a bool

so

assuming args[0] = '' which is trucy false (because a empty string is default trucy false)
doing the following now !!args[0] will convert the return value to a boolean even though its a string

Reply:
it's basically an easy way to check if there is anything at args[0] or [1] but I realised it'll be easier and more readable to just check the length of args

Reply:
```javascript
args = ["af"]
!!args[0]
> true
!!args[1]
> false
```